### PR TITLE
Enhancement to Alarm Page + Backend

### PR DESCRIPTION
**Pull request by JingtingL**
Unit Testing and Enhancement to the frontend of alarm page

### CODE REVIEW

File: client/src/components/elements/AlarmModal.jsx
Comment (Shuzhengz) [inline client/src/components/elements/AlarmModal.jsx:1]: maybe we shoul change it to `.tsx`

File: client/src/components/pages/alarm.tsx [hunk 1]
```diff
@@ -18,26 +18,7 @@ type AlarmType = {
```
Comment (Shuzhengz) [inline client/src/components/pages/alarm.tsx:18]: add back:

```typescript
useEffect(() => {
        document.title = "Sleep Buddies - Alarms";
    }, []);
```

**Review by Shuzhengz — CHANGES_REQUESTED** [review]

File: client/src/components/pages/alarm.tsx (hunk 1 above)
Comment (JingtingL) [reply client/src/components/pages/alarm.tsx:18]: it's there?

Comment (JingtingL) [reply client/src/components/pages/alarm.tsx:18]: image

File: client/src/components/elements/AlarmModal.jsx
Comment (JingtingL) [reply client/src/components/elements/AlarmModal.jsx:1]: Done :)

File: client/src/components/pages/alarm.tsx (hunk 1 above)
Comment (Shuzhengz) [reply client/src/components/pages/alarm.tsx:18]: ah i didnt see it lol

**Pull request merged**